recipe/meta.yaml

```diff
@@ -117,7 +92,6 @@ outputs:
     requirements:
```

Comment:
probably this could also be handled better, although maybe better to do this when there's a -gpu variant